tests/support_server/setup.pm

```diff
@@ -314,42 +314,62 @@ sub setup_iscsi_server {
     # The easiest way (really!?) to configure LIO is with YaST
     # Code grab and adapted from tests/iscsi/iscsi_server.pm
     script_run("yast2 iscsi-lio-server; echo yast2-iscsi-lio-server-status-\$? > /dev/$serialdev", 0);
-    assert_screen 'iscsi-target-overview-service-tab';
-    send_key 'alt-g';    # go to global tab
-    assert_screen 'iscsi-target-overview-global-tab';
+    assert_screen 'iscsi-target-overview-service-tab', 60;
     send_key 'alt-t';    # go to target tab
-    wait_still_screen 3;
+    assert_screen 'iscsi-target-overview-empty-target-tab';
     send_key 'alt-a';    # add target
-    wait_still_screen 3;
-    send_key 'alt-t';    # select target field
-    wait_still_screen 3;
-    for (1 .. 40) { send_key 'backspace'; }    # delete text
+    assert_screen 'iscsi-target-overview-add-target-tab';
+
+    # Wait for the Identifier field to change from 'test' value to the correct one
+    # We could simply use a 'sleep' here but it's less good
+    wait_screen_change(undef, 10);
+
+    # Select Target field
+    send_key 'alt-t';
+    wait_still_screen;
```

Comment:
standard wait_still_screen waits for 7 seconds of still screen. Can this be reduced a bit?

Reply:
3 seconds was too small during my tests on ppc64le, I will try with 5 seconds to see if it's ok.

Reply:
After some mew tests it appears that 3 seconds is now ok... I think that the first issue was on the "Global" iSCSI screen, and I removed it after because it is not necessary in the case of iSCSI LIO configuration.

So, I changed the code to revert back to 3 seconds.